ignite/services/chain/chain.go

```diff
@@ -44,7 +44,7 @@ type Chain struct {
 
 	Version cosmosver.Version
 
-	plugin         Plugin
+	Plugin         Plugin
```

Comment:
What is a `Plugin` in that context ? Seems like an abstracted way to interact with a chain, but there's currently a single implementation.

Reply:
Yes, its basically a way we interact with chain config files (`app.toml`, `config.toml`, `client.toml`...).  The `plugin` name does not make much sense to me at all.

Reply:
same ^^'

Reply:
We used to have this to support different Cosmos SDK "distribution" (launchpad and stargate), so you have a "plugin" to interact with the config of a specific version but now we only Stargate and yes Plugin naming can be misleading even more that now we have plugins for the CLI